src/main.rs

```diff
@@ -310,6 +317,38 @@ async fn run_server(addr: SocketAddr) -> anyhow::Result<()> {
     Ok(())
 }
 
+/// Spawns a background tokio task which runs all jobs having no schedule
+/// i.e. manually started
+fn spawn_job_oneoffs(ctx: Arc<Context>) {
+    // TODO: Ideally JobSchedule.schedule should become an `Option<Schedule>`
+    // and here we run all those with schedule=None
```

Comment:
@jackh726 annoted here your suggestion on how to run non-scheduled job (for a separate patch)

Reply:
Hmm, not quite what I was thinking, but this is good!